app/controllers/assignments_controller.rb

```diff
@@ -211,7 +216,8 @@ def assignment_params
                                ],
                                graders_attributes: [
                                  :avail_score, :upload_file, :params,
-                                 :type, :id, :_destroy
+                                 :type, :id, :_destroy, :errors_to_show, :test_class,
```

Comment:
It might be possible to eliminate that permission from here, since it gets unconditionally set on line 116.  I had thought it was necessary or else the strong-params object might not permit the additional key.  If it's not needed, I'm happy to eliminate it.